sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDao.java

```diff
@@ -267,6 +329,10 @@ public void deleteUserPermission(DbSession session, Long templateId, Long userId
     session.commit();
   }
 
+  /**
+   * @deprecated since 5.2 use {@link #insertGroupPermission(DbSession, Long, Long, String)}
+   */
+  @Deprecated
   public void insertGroupPermission(Long templateId, @Nullable Long groupId, String permission) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)